### PR TITLE
update the markdown file in the glossary directory for Chinese version.

### PR DESCRIPTION
**Pull request by jialaijun**
update the markdown file in the glossary directory for Chinese version.

### CODE REVIEW

File: content/zh/docs/reference/glossary/kube-scheduler.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ id: kube-scheduler
 date: 2018-04-12
 full_link: /docs/reference/generated/kube-scheduler/
 short_description: >
-  主节点上的组件，该组件监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
+  控制平面组件监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
```
Comment (SataQiu) [inline content/zh/docs/reference/glossary/kube-scheduler.md:7]:
```suggestion
  控制平面组件，监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
```

File: content/zh/docs/reference/glossary/kube-scheduler.md [hunk 2]
```diff
@@ -28,10 +28,12 @@ tags:
...
 <!--
- Component on the master that watches newly created pods that have no node assigned, and selects a node for them to run on.
--->
+Control plane component that watches for newly created
+{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
+{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
+to run on.-->
 
-主节点上的组件，该组件监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
+控制平面组件监视那些新创建的未指定运行{{< glossary_tooltip term_id="node" text="节点（node）">}}的 {{< glossary_tooltip term_id="pod" text="Pods" >}}，并选择节点让 Pod 在上面运行。
```
Comment (SataQiu) [inline content/zh/docs/reference/glossary/kube-scheduler.md:36]:
```suggestion
控制平面组件，监视那些新创建的未指定运行{{< glossary_tooltip term_id="node" text="节点（node）">}}的 {{< glossary_tooltip term_id="pod" text="Pods" >}}，并选择节点让 Pod 在上面运行。
```

File: content/zh/docs/reference/glossary/mirror-pod.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 静态 Pod
+title: Mirror Pod
```
Comment (SataQiu) [inline content/zh/docs/reference/glossary/mirror-pod.md:2]: “Mirror Pod” 与下文的 ”静态 Pod“  不一致，建议前后保持一致。

File: content/zh/docs/reference/glossary/kube-scheduler.md (hunk 1 above)
Comment (tengqm) [inline content/zh/docs/reference/glossary/kube-scheduler.md:7]:
```suggestion
  控制平面组件，负责监视新创建的、未指定运行节点的 Pod，选择节点让 Pod 在上面运行。
```

File: content/zh/docs/reference/glossary/kube-scheduler.md (hunk 2 above)
Comment (tengqm) [inline content/zh/docs/reference/glossary/kube-scheduler.md:36]: 同上

File: content/zh/docs/reference/glossary/minikube.md [hunk 4]
```diff
@@ -38,6 +38,10 @@ Minikube 是用来在本地运行 Kubernetes 的一种工具。
 
 <!--
 Minikube runs a single-node cluster inside a VM on your computer.
+You can use Minikube to
+[try Kubernetes in a learning environment](/docs/setup/learning-environment/).
 -->
 
 Minikube 在用户计算机上的一个虚拟机内运行单节点 Kubernetes 集群。
+你可以使用 Minikube 
+[在学习环境中尝试 Kubernetes](/docs/setup/learning-environment/).
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/minikube.md:47]:
```suggestion
[在学习环境中尝试 Kubernetes](/zh/docs/setup/learning-environment/).
```

File: content/zh/docs/reference/glossary/mirror-pod.md [hunk 5]
```diff
@@ -43,4 +44,4 @@ will be visible on the API server, but cannot be controlled from there.
 它会自动地尝试在 Kubernetes API 服务器上为它创建 Pod 对象。
 这意味着 pod 在 API 服务器上将是可见的，但不能在其上进行控制。
 
-（例如，删除静态 pod 将不会停止 kubelet 守护程序的运行）。
+（例如，删除静态 pod 将不会停止由 kubelet 启动运行的守护程序）。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/mirror-pod.md:47]:
```suggestion
（例如，删除镜像 Pod 也不会阻止 kubelet 守护进程继续运行它）。
```

File: content/zh/docs/reference/glossary/mirror-pod.md [hunk 6]
```diff
@@ -27,7 +27,8 @@ tags:
  A {{< glossary_tooltip text="pod" term_id="pod" >}} object that a kubelet uses
  to represent a {{< glossary_tooltip text="static pod" term_id="static-pod" >}}
 -->
- kubelet 使用一个对象 {{< glossary_tooltip text="pod" term_id="pod" >}} 来代表 {{< glossary_tooltip text="static pod" term_id="static-pod" >}}
+ Mirror Pod 是一个被 kubelet 用来代表 {{< glossary_tooltip text="static pod" term_id="static-pod" >}} 的
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/mirror-pod.md:30]:
```suggestion
镜像 Pod（Mirror Pod）是被 kubelet 用来代表{{< glossary_tooltip text="静态 Pod" term_id="static-pod" >}} 的
```

File: content/zh/docs/reference/glossary/mirror-pod.md (hunk 3 above)
Comment (tengqm) [inline content/zh/docs/reference/glossary/mirror-pod.md:2]:
```suggestion
title: 镜像 Pod（Mirror Pod）
```

File: content/zh/docs/reference/glossary/namespace.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 命名空间
+title: 命名空间（Namespace）
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/namespace.md:2]:
```suggestion
title: 名字空间（Namespace）
```

Comment (tengqm) [reply content/zh/docs/reference/glossary/namespace.md:2]: 请将此词条中“命名空间”改为“名字空间”.

File: content/zh/docs/reference/glossary/object.md [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
-title: 对象
+title: 对象（Object）
 id: object
 date: 2020-10-12
-full_link: https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects
+full_link: https://kubernetes.io/zh/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/object.md:5]:
```suggestion
full_link: /zh/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects
```

File: content/zh/docs/reference/glossary/object.md [hunk 9]
```diff
@@ -37,6 +37,6 @@ By creating an object, you're effectively telling the Kubernetes system what you
 your cluster's workload to look like; this is your cluster's desired state.
 -->
 Kubernetes 对象通常是一个“目标记录”-一旦你创建了一个对象，Kubernetes 
-{{< glossary_tooltip text="控制平面" term_id="control-plane" >}} 
+{{< glossary_tooltip text="控制平面（control plane）" term_id="control-plane" >}}
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/object.md:40]:
```suggestion
{{< glossary_tooltip text="控制平面（Control Plane）" term_id="control-plane" >}} 
```

File: content/zh/docs/reference/glossary/persistent-volume-claim.md [hunk 10]
```diff
@@ -1,8 +1,8 @@
 ---
-title: 持久卷申领
+title: 持久卷申领（Persistent Volume Claim）
 id: persistent-volume-claim
 date: 2018-04-12
-full_link: /docs/concepts/storage/persistent-volumes/
+full_link: /zh/docs/concepts/storage/persistent-volumes/
```
Comment (SataQiu) [inline content/zh/docs/reference/glossary/persistent-volume-claim.md:5]: 这里是否需要添加 /zh 前缀？不加 /zh 似乎也不影响页面浏览？
而且与之前的修改相矛盾，到底是去 /zh 还是加 /zh ？

**Review by SataQiu — COMMENTED** [review]
感谢 @jialaijun  贡献，不过存在一些小问题，做了备注！

File: content/zh/docs/reference/glossary/persistent-volume-claim.md (hunk 10 above)
Comment (jialaijun) [reply content/zh/docs/reference/glossary/persistent-volume-claim.md:5]: 因为是中文文档，所以对应的连接也最好是中文版本的。本文件夹下都是参照此规范进行的。

File: content/zh/docs/reference/glossary/kube-scheduler.md (hunk 1 above)
Comment (jialaijun) [reply content/zh/docs/reference/glossary/kube-scheduler.md]: yeah，i have made a modification.

File: content/zh/docs/reference/glossary/kube-scheduler.md [hunk 11]
```diff
@@ -28,10 +28,12 @@ tags:
...
 <!--
- Component on the master that watches newly created pods that have no node assigned, and selects a node for them to run on.
--->
+Control plane component that watches for newly created
+{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
+{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
+to run on.-->
 
-主节点上的组件，该组件监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
+  控制平面组件，负责监视新创建的、未指定运行{{< glossary_tooltip term_id="节点（node）" text="node">}}的 {{< glossary_tooltip term_id="pod" text="Pods" >}}，选择节点让 Pod 在上面运行。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kube-scheduler.md:36]: you cannot change the term_id. You can change the text if you like. Ie

```suggestion
控制平面组件，负责监视新创建的、未指定运行{{< glossary_tooltip term_id="node" text="节点（node）">}}的
{{< glossary_tooltip term_id="pod" text="Pods" >}}，选择节点让 Pod 在上面运行。
```

Comment (jialaijun) [reply content/zh/docs/reference/glossary/kube-scheduler.md]: yes,I have updated the entry justly.

File: content/zh/docs/reference/glossary/kubernetes-api.md [hunk 12]
```diff
@@ -17,7 +17,7 @@ tags:
 title: Kubernetes API
 id: kubernetes-api
 date: 2018-04-12
-full_link: /zh/docs/concepts/overview/kubernetes-api/
+full_link: /docs/concepts/overview/kubernetes-api/
```
Comment (sftim) [inline content/zh/docs/reference/glossary/kubernetes-api.md:20]: Why this change?

File: content/zh/docs/reference/glossary/name.md [hunk 13]
```diff
@@ -16,7 +16,7 @@ tags:
 title: Name
 id: name
 date: 2018-04-12
-full_link: /zh/docs/concepts/overview/working-with-objects/names/
+full_link: /docs/concepts/overview/working-with-objects/names/
```
Comment (sftim) [inline content/zh/docs/reference/glossary/name.md:19]: Why this change?

File: content/zh/docs/reference/glossary/network-policy.md [hunk 14]
```diff
@@ -18,7 +18,7 @@ tags:
 title: Network Policy
 id: network-policy
 date: 2018-04-12
-full_link: /zh/docs/concepts/services-networking/network-policies/
```
Comment (sftim) [inline content/zh/docs/reference/glossary/network-policy.md:21]: Why this change?

File: content/zh/docs/reference/glossary/operator-pattern.md [hunk 15]
```diff
@@ -15,7 +15,7 @@ tags:
 title: Operator pattern
 id: operator-pattern
 date: 2019-05-21
-full_link: /zh/docs/concepts/extend-kubernetes/operator/
+full_link: /docs/concepts/extend-kubernetes/operator/
```
Comment (sftim) [inline content/zh/docs/reference/glossary/operator-pattern.md:18]: Why this change?

File: content/zh/docs/reference/glossary/pod-security-policy.md [hunk 16]
```diff
@@ -17,7 +17,7 @@ tags:
 title: Pod Security Policy
 id: pod-security-policy
 date: 2018-04-12
-full_link: /zh/docs/concepts/policy/pod-security-policy/
+full_link: /docs/concepts/policy/pod-security-policy/
```
Comment (sftim) [inline content/zh/docs/reference/glossary/pod-security-policy.md:20]: Why this change?

**Review by sftim — COMMENTED** [review]
I have a query about 5 regressions that I spotted.

File: content/zh/docs/reference/glossary/kubernetes-api.md (hunk 12 above)
Comment (tengqm) [reply content/zh/docs/reference/glossary/kubernetes-api.md:20]: It is the original English version, commented out.

File: content/zh/docs/reference/glossary/name.md (hunk 13 above)
Comment (sftim) [reply content/zh/docs/reference/glossary/name.md:19]: It is the original English version, commented out.

File: content/zh/docs/reference/glossary/network-policy.md (hunk 14 above)
Comment (sftim) [reply content/zh/docs/reference/glossary/network-policy.md:21]: It is the original English version, commented out.

File: content/zh/docs/reference/glossary/operator-pattern.md (hunk 15 above)
Comment (sftim) [reply content/zh/docs/reference/glossary/operator-pattern.md:18]: It is the original English version, commented out.

File: content/zh/docs/reference/glossary/pod-security-policy.md (hunk 16 above)
Comment (sftim) [reply content/zh/docs/reference/glossary/pod-security-policy.md:20]: It is the original English version, commented out.

File: content/zh/docs/reference/glossary/kubernetes-api.md (hunk 12 above)
Comment (jialaijun) [reply content/zh/docs/reference/glossary/kubernetes-api.md:20]: It is the original English version, commented out. @sftim

File: content/zh/docs/reference/glossary/name.md (hunk 13 above)
Comment (jialaijun) [reply content/zh/docs/reference/glossary/name.md:19]: > Why this change?

It is the original English version, commented out.

File: content/zh/docs/reference/glossary/kubernetes-api.md [hunk 17]
```diff
@@ -17,7 +17,7 @@ tags:
 title: Kubernetes API
 id: kubernetes-api
 date: 2018-04-12
-full_link: /docs/concepts/overview/kubernetes-api/
+full_link: /zh/docs/concepts/overview/kubernetes-api/
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kubernetes-api.md]: Why change the commented URL to zh?

Comment (jialaijun) [reply content/zh/docs/reference/glossary/kubernetes-api.md]: i have commited right edition justly.
